packages/aws-cdk/lib/init-templates/v1/app/fsharp/add-project.hook.ts

```diff
@@ -1,6 +1,6 @@
 import * as child_process from 'child_process';
 import * as path from 'path';
-import { InvokeHook } from '../../../init';
+import { InvokeHook } from '../../../../init';
```

Comment:
why is the `init.ts` also not moved into the `/v1` folder?

Reply:
Because (for now), it's absolutely identical between v1 and v2. It's the same "init" in this PR (that you reviewed). I didn't see any reason to fraction the whole `lib` directory between v1 and v2, just the templates themselves.